patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback-success.twig

```diff
@@ -0,0 +1,15 @@
+<div class="ma__mass-feedback-form ma__mass-feedback-form--success" id="feedback" data-mass-feedback-form>
+<form class="ma__mass-feedback-form__form" method="post" novalidate="" action="{{action}}" class="" id="">
```

Comment:
Maybe we should have options to pass in class or id?  Maybe even make novalidate optional based on a flag?